recipes/semvecpy/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "semvecpy" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 10c62aab3a8bb387f1f9b132b74ea266d2331c0a05fce5b6f911b2972e08dfd9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
```

Could you please change it to be ``noarch``?
You can select python on host and run

```suggestion
  noarch: python
```